src/app/Scenes/MyCollection/Components/MyCollectionStickyHeader.tsx

```diff
@@ -218,7 +215,7 @@ const AnimatedPill = ({
   }
 
   const selected = selectedTab === tab
-  const disabled = !!selectedTab && selectedTab !== tab
+  const pressable = (!!selectedTab && selectedTab !== tab) || disabled
```

Comment:
**suggestion:** Shouldn't this be the opposite? Right now the pill is pressable if it's disabled 🤔

Reply:
trueee 😄 do you have a name you suggest

Reply:
Hmm, wither passing `!!selectedTab && selectedTab !== tab` directly as a prop or something like:

```suggestion
  const disabled = (!!selectedTab && selectedTab !== tab) || disabledProp
```

Reply:
good suggestion - i will change